pkgs/applications/misc/emem/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "emem";
-  version = "0.2.41";
+  version = "0.2.42";
   name = "${pname}-${version}";
 
   inherit jdk;
 
   src = fetchurl {
```

Comment:
Okay! However, I don’t need the .tar.gz file. It’s the .jar file, specifically, that I need.